packages/sdk-components-react/src/time.ws.ts

```diff
@@ -0,0 +1,33 @@
+import { TimeIcon } from "@webstudio-is/icons/svg";
+
+import {
+  defaultStates,
+  type PresetStyle,
+  type WsComponentMeta,
+  type WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+
+import { time } from "@webstudio-is/react-sdk/css-normalize";
+
+import { props } from "./__generated__/time.props";
+
+const presetStyle = {
+  time: [...time],
```

Comment:
why spread?

Reply:
copypasted, in case of any other styles will be added